### PR TITLE
COMMERCE-6405 Commerce media resolver vulnerability issue

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-media-impl/src/main/java/com/liferay/commerce/media/internal/DefaultCommerceMediaResolver.java [hunk 1]
```diff
@@ -139,8 +187,27 @@ public String getUrl(
 
 		if (secure) {
 			cpAttachmentFileEntry =
-				_cpAttachmentFileEntryLocalService.fetchCPAttachmentFileEntry(
+				_cpAttachmentFileEntryLocalService.getCPAttachmentFileEntry(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-media-impl/src/main/java/com/liferay/commerce/media/internal/DefaultCommerceMediaResolver.java:190]: move out of if statement

Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-media-impl/src/main/java/com/liferay/commerce/media/internal/DefaultCommerceMediaResolver.java:190]: use fetch instead

File: modules/apps/commerce/commerce-media-test/src/testIntegration/java/com/liferay/commerce/media/test/CommerceMediaResolverTest.java [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+			RandomTestUtil.randomString() + "@liferay.com",
+			RandomTestUtil.randomString(), _serviceContext);
+
+		CommerceAccountTestUtil.addCommerceAccountGroupAndAccountRel(
+			_user.getCompanyId(), RandomTestUtil.randomString(),
+			CommerceAccountConstants.ACCOUNT_GROUP_TYPE_STATIC,
+			_commerceAccount.getCommerceAccountId(), _serviceContext);
+	}
+
+	@Test
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-media-test/src/testIntegration/java/com/liferay/commerce/media/test/CommerceMediaResolverTest.java:107]: remove CDN refence

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:stop

**Comment by Commando18** [comment]
ci:stop

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**